pallets/protos/src/lib.rs

```diff
@@ -821,79 +838,61 @@ pub mod pallet {
 			if let Some(struct_proto) = <Protos<T>>::get(proto_id) {
 				if let Some(avail) = avail {
 					if avail && struct_proto.include_cost.is_none() {
-						return false
+						return false;
 					} else if !avail && struct_proto.include_cost.is_some() {
-						return false
+						return false;
 					}
 				}
 
 				if categories.len() == 0 {
-					return Self::filter_tags(tags, &struct_proto)
-				} else {					
-					return Self::filter_category(tags, &struct_proto, categories)
+					return Self::filter_tags(tags, &struct_proto);
+				} else {
+					return Self::filter_category(tags, &struct_proto, categories);
```

Comment:
also here, trick is you could have removed `return` and in that case no need for semicolon

Reply:
Yeah, right! Sorry for this!